src/App.jsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { ApplicationInsights, SeverityLevel } from '@microsoft/applicationinsights-web';
+import { SeverityLevel } from '@microsoft/applicationinsights-web';
+import { ReactAI, withAITracking } from "react-appinsights";
+import { Router } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
 import './App.css';
 
+const history = createBrowserHistory()
 let INSTRUMENTATION_KEY = '4a795cb3-5b9e-4428-8777-0441b7ae7dc8'; // Enter your instrumentation key here
 
-let appInsights = new ApplicationInsights({config: {
+ReactAI.initialize({
```

Comment:
In terms of work done by initialize, this method is essentially initializing the Web SKU with inputs and doing the react-appinsights initialization. What this is also doing is removing the ability for customer to use any other application insights extension, which does not seem necessary. Looking at the beta code, rootInstance is returning the ai instance initialized.

Here is a list of changes that you would need to do:
1. Add telemetryinitializer to update context properties. It seems you would like to the whole context setup, I can look into adding a public property for this on the properties extension so you can continue with you code as before.
2. You already have customDimensioninitializer that is essentially taking advantage of telemetryInitializer.

Let me know if you need help updating to implement ITelemetryPlugin, happy to setup some time for this.